### ISSUE TITLE
Test report styling cannot handle message arguments in test results with backslashes 

### ISSUE THREAD

**Issue by cportele** [interactive-instruments/etf-webapp#102]
See https://github.com/inspire-eu-validation/ets-repository/issues/52

In this case:

```xml
<message ref="TR.urlNotHttp">
<translationArguments>
<argument token="filename">nlps_nationaleparken.gml.xml</argument>
<argument token="id">dee67ed2-ace0-46ec-a81c-36cd29448940</argument>
<argument token="url">
\\lnv.intern\grp\RVO\GCC\Beheer\_Databeheer\_INSPIRE\Data\Beschermde gebieden\Geharmoniseerde INSPIRE data\Nationale_Parken\nlps_nationaleparken.gml
</argument>
</translationArguments>
</message>
```

Maybe other characters need special treatment as well?

**Comment by jonherrmann** [comment]
Issue has been fixed in version 2.0.0 build 1703xx.

**Comment by jonherrmann** [comment]
For the described case "/", it is already fixed. Added code for the second case when a $ literal is in the argument string.

https://www.w3.org/TR/xpath-functions/#func-replace
